src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_TupleCreationExpression.cs

```diff
@@ -46,25 +51,25 @@ private BoundNode RewriteTupleCreationExpression(BoundTupleExpression node, Immu
                 var smallestCtor = (MethodSymbol)TupleTypeSymbol.GetWellKnownMemberInType(smallestType.OriginalDefinition,
                                                                                             TupleTypeSymbol.GetTupleCtor(smallestType.Arity),
                                                                                             _diagnostics,
-                                                                                            node.Syntax);
+                                                                                            syntax);
                 if ((object)smallestCtor == null)
                 {
-                    return node;
+                    return null;
```

Comment:
I think we shouldn't return null from this function. We can create and return a bad node though.

---
In reply to: [65586552](https://github.com/dotnet/roslyn/pull/11681#discussion_r65586552) [](ancestors = 65586552)